site/content/docs/04-cli-api.md

```diff
@@ -0,0 +1,46 @@
+---
+title: CLI and Programmatic API
+---
+
+Sapper ships a small CLI that you can use to build, serve, and export your Sapper apps:
+
+- `sapper dev`: spin up a local dev server locally with hot reloading
+- `sapper build`: build a Sapper app that can be deployed on a Node server
+- `sapper export`: export a Sapper app as a static site, prefetching data
+
+You may call these via API or programmatically by importing the API (check the Sapper source for details on this).
+
+In the CLI, you can pass flags to each command, e.g. `sapper dev --legacy --ext ".svelte .svexy"`. Here is a full list of common API flags/options and what they do:
```

Comment:
I do love mdsvex's default file extension, but this feels a little unprofessional lol. Maybe a different example?

Reply:
will use `.html`

Reply:
I beg your very pardon!